parser-typechecker/src/Unison/TypeParser.hs

```diff
@@ -84,9 +84,9 @@ effectList = do
 
 sequenceTyp :: Var v => TypeP v
 sequenceTyp = do
-  open <- reserved "["
+  open <- openBlockWith "["
```

Comment:
Maybe it's a bit weird that blocks can appear in list types, but there aren't any block type elements at this level anyway.